src/components/menu-component.jsx

```diff
@@ -104,10 +124,27 @@ class MainMenu extends React.Component {
 
 
   render() {
+    const actions = [
+      <FlatButton
+        label="Cancel"
+        primary
+        onClick={this.switchModal}
+      />,
+      <FlatButton
+        label="Submit"
+        primary
+        onClick={this.closeModalAndDeleteNotebook}
+      />,
+    ];
     const mc = menuComponents(menuItems, this)
 
     return (
       <ToolbarGroup firstChild={this.props.firstChild}>
+        <Dialog
+          open={this.state.isModalOpen}
+          title="Are you sure you want to delete this file?"
```

Comment:
- Could you remove the text from the `title` and put it in line between `<Dialog etc...> {text here} </Dialog>`? This doesn't need a title.
- The text should read "Delete notebook?"